components/home.js

```diff
@@ -1,49 +1,132 @@
 import Carousel, { ChannelCarousel } from './carousel'
 import { useDNav } from './dnav'
 
-import { useFavoritesStorage, useRecentsStorage } from '../src/storage'
+import { useFavoritesStorage, useRecentsStorage, useWizardStorage } from '../src/storage'
+import { useITunesData } from '../pages/api/itunes'
+import { useYouTubeData } from '../pages/api/youtube'
+import Layout from '../components/layout'
+import { LoadingMessage } from '../components/loading'
+import Error from '../components/error'
+import { useRouter } from './vlink'
 
 import styles from '../styles/Home.module.css'
 
 const FAVORITES_HELP = (
-  <small>Subscribe to channels you like and they'll show up here</small>
+  <small>Subscribe to channels you like and they&apos;ll show up here</small>
 )
 
-export default function Home () {
-  const favorites = useFavoritesStorage()
+const CATEGORIES = ['News', 'Sport', 'Comedy', 'Cars', 'Music']
+
+function SearchCarousel ({ response }) {
+  const { data, error } = response || {}
+
+  if (error) {
+    console.error(error)
```

Comment:
I _think_ `console.error` didn't existing on my TV when I was testing yesterday. (I know this pattern is repeated elsewhere... probably have to clean that up)